package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "authors-haven",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "scripts": {
+    "build": "NODE_ENV=production webpack --config ./config/webpack.prod.js",
+    "lint": "npx eslint ./src/**.js --fix",
+    "test": "jest",
+    "dev": "NODE_ENV=development webpack-dev-server --open --config ./config/webpack.dev.js"
+  },
+  "author": "Mazus",
+  "license": "MIT",
+  "dependencies": {
+    "@babel/polyfill": "7.2.5",
+    "prop-types": "15.6.2",
+    "react": "16.8.1",
+    "react-dom": "16.8.1"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.2.3",
+    "@babel/core": "7.2.2",
+    "@babel/plugin-proposal-class-properties": "7.2.3",
+    "@babel/preset-env": "7.2.3",
+    "@babel/preset-react": "7.0.0",
+    "autoprefixer": "9.4.6",
+    "babel-eslint": "10.0.1",
+    "babel-loader": "8.0.5",
+    "clean-webpack-plugin": "1.0.0",
+    "css-loader": "2.1.0",
+    "cssnano": "4.1.8",
+    "dotenv": "^8.0.0",
+    "eslint": "5.12.0",
+    "eslint-config-airbnb": "17.1.0",
+    "eslint-loader": "2.1.1",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-jsx-a11y": "6.1.2",
+    "eslint-plugin-react": "7.12.3",
+    "eslint-plugin-react-hooks": "1.6.0",
+    "file-loader": "3.0.1",
+    "friendly-errors-webpack-plugin": "1.7.0",
+    "html-webpack-plugin": "3.2.0",
+    "mini-css-extract-plugin": "0.5.0",
+    "optimize-css-assets-webpack-plugin": "5.0.1",
+    "postcss-loader": "3.0.0",
+    "react-hot-loader": "4.6.3",
+    "react-test-renderer": "16.7.0",
+    "style-loader": "0.23.1",
+    "uglifyjs-webpack-plugin": "2.1.1",
+    "webpack": "4.28.4",
+    "webpack-cli": "3.2.1",
+    "webpack-dev-server": "3.1.14",
+    "webpack-merge": "4.2.1"
+  }
+}
```

Comment:
All these plenty dependencies.. 👀

Reply:
@Tunji17 please squash your commits

Reply:
I have done that, I was having issues with my internet and had to go to IT

Reply:
Yes @Tunji17, you have 1 commit now. But still, peep the commit message and _refactor_ 
<img width="946" alt="Screenshot 2019-08-26 at 6 43 35 PM" src="https://user-images.githubusercontent.com/15054956/63710790-e99f6180-c831-11e9-9f67-3509b1cc4417.png">

Reply:
@mazma1  I have refactored the commit message